hw/src/ledger.rs

```diff
@@ -363,7 +364,7 @@ impl Manager {
 }
 
 // Try to connect to the device using polling in at most the time specified by the `timeout`
-fn try_connect_polling(ledger: Arc<Manager>, timeout: &Duration, device_direction: DeviceDirection) -> bool {
+fn try_connect_polling(ledger: &Arc<Manager>, timeout: &Duration, device_direction: DeviceDirection) -> bool {
```

Comment:
maybe just `&Manager`? `&Arc<Manager>` will be automatically dereferenced to it

Reply:
Goood catch thanks :1st_place_medal: